react/features/base/aspect-ratio/actions.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+
+import { SET_ASPECT_RATIO } from './actionTypes';
+import { ASPECT_RATIO_NARROW, ASPECT_RATIO_WIDE } from './constants';
+
+/**
+ * Calculates new aspect ratio for the app based on provided width and height
+ * values.
+ *
+ * @param {number} width - The width of the app's area used on the screen.
+ * @param {number} height - The height of the app's area used on the screen.
+ * @returns {{
+ *      type: SET_ASPECT_RATIO,
+ *      aspectRatio: Symbol
+ * }}
+ */
+export function calculateNewAspectRatio(width: number, height: number): Object {
```

Comment:
why not call this `setAspectRatio` and do the calculation outside of this function?